### ISSUE TITLE
Nested Menu Structure

### ISSUE THREAD

**Issue by bbischler** [ookami-kb/storybook_flutter#53]
Hi @ookami-kb,

first of all thanks for your work!

It would be nice to have something like _subsections_ for the menu.

Example: 

- Basic Components
- - Buttons
- - - Primary Button
- - - Secundary Button
- - etc.

**Comment by ookami-kb** [comment]
Hi, thanks!

Not sure how to do it in a nice and user-friendly way, but let me think about it.

**Comment by bbischler** [comment]
- Maybe introduce a `subSection` property on the Story-Widget.
- Or allow to specifiy the structure with `/` in the `section`-property. like so:
   ```
   Story(
    section: 'Basic Components/Buttons',
  )
   ```

**Comment by ookami-kb** [comment]
Yeah, the second option sounds good. 

🤔 Maybe we can even merge `name` and `section`, and allow some custom structure, e.g.

```
Basic Components/Buttons/Primary Button
Basic Components/Input
```

**Comment by fel-cesar** [comment]
I love that Idea!

**Comment by aaisataev** [comment]
Hi guys,
Any news on this? 🙏

**Comment by ookami-kb** [comment]
Actually, it should be resolved in #80
